File: ServiceRequest/DTOs/ServiceRequestDTO.cs
using ServiceRequest.Models.Enums;

namespace ServiceRequest.DTOs
{
    public class ServiceRequestDTO
    {
        public string? Title { get; set; } 
        public string? Description { get; set; }
        public RequestType RequestType { get; set; }
        public Priority Priority { get; set; }
        public string? CreatedBy { get; set; }

    }

}

File: FrontEnd/service-request/src/app/models/DTOs/service-request-dto.ts
import { Priority } from '../Enums/priority.enum';
import { RequestType } from '../Enums/request-type.enum';  
import { Status } from '../Enums/status.enum';


export interface ServiceRequestDto {
  title: string;
  description?: string;
  requestType: RequestType;
  priority: Priority;
  createdBy: string;
  attachments?: File[];
}
